components/schedule-editor/src/ScheduleEditor.svelte

```diff
@@ -519,676 +540,705 @@
     bind:clientWidth={mainElementWidth}
     bind:this={main}>
     <div class="settings">
-      <nylas-schedule-editor-section
-        section_title={SectionNames.BASIC_DETAILS}
-        expanded={true}>
-        <h1 slot="title">Event Details</h1>
-        <p slot="intro" class="intro">
-          Edit the details for your meeting. You can add consecutive meetings to
-          allow users to book back-to-back events.
-        </p>
-        <div slot="contents" class="contents basic-details">
-          {#each _this.events as event, iter}
-            <fieldset>
-              {#if _this.events.length > 1}
-                <button
-                  class="remove-event"
-                  on:click={() => removeConsecutiveEvent(event)}>
-                  Remove this event
-                </button>
-              {/if}
-              <label>
-                <strong>Event Title!</strong>
-                <input type="text" bind:value={event.event_title} />
-              </label>
-              <label>
-                <strong>Event Description</strong>
-                <input type="text" bind:value={event.event_description} />
-              </label>
-              <label>
-                <strong>Event Location</strong>
-                <input type="text" bind:value={event.event_location} />
-              </label>
-              <label>
-                <strong>Conferencing Link (Zoom, Teams, or Meet URL)</strong>
-                <input type="url" bind:value={event.event_conferencing} />
-              </label>
-              <div role="radiogroup" aria-labelledby="slot_size">
-                <strong id="slot_size">Meeting Length</strong>
+      {#if _this.sections[SectionNames.BASIC_DETAILS].editable}
```

Comment:
Review with whitespace hidden for your own sanity, plz. Some sub-contents here went over the line length threshold, hence the large number of changes.

![image](https://user-images.githubusercontent.com/713991/157116346-fe57b24a-0343-48f1-89f6-491072166f0c.png)